components/src/NavBar/DesktopMenu.js

```diff
@@ -6,29 +6,73 @@ import MenuTrigger from "./MenuTrigger";
 import MenuLink from "./MenuLink";
 import theme from "../theme";
 
-const isTrigger = menuItem => (menuItem.items);
+const ApplyMenuLinkStyles = styled.div({
+  "& *": {
```

Comment:
i see these in a number of places. does e.g `& *` do anything vs just `*`? i'm pretty sure `& ` renders to nothing. like `li &:hover` would join the parent to be `li:hover`, so `li & *` would join the parent to just be `li *`.

Reply:
Will make these changes, & will render to nothing while not including & will render to a space with basic nesting (hence why it is needed in the li &:hover example but not in these cases.